src/com/google/enterprise/adaptor/MetadataTransform.java

```diff
@@ -25,6 +25,7 @@
  */
 public interface MetadataTransform {
 
+  /** Keys for data stored in the {@code Map} of {@code params}. */
```

Comment:
Change to a // or /* comment, since this would be completely misleading to read in javadoc.